packages/components/src/organisms/ShippingSimulation/ShippingSimulation.tsx

```diff
@@ -0,0 +1,184 @@
+import type { HTMLAttributes } from 'react'
+import type { PriceFormatter } from '../../atoms/Price/Price'
+
+import React from 'react'
+
+import { ArrowSquareOut } from '../../assets'
+
+import {
+  Icon,
+  Link,
+  Table,
+  TableBody,
+  TableCell,
+  TableRow,
+  InputField,
+  Price,
+  LinkProps,
+  LinkElementType,
+} from '../..'
+
+interface ShippingSla {
```

Comment:
Can this interface be called `ShippingSLA`?

Reply:
Sounds better hahah.